addon/services/file-manager.js

```diff
@@ -141,25 +244,65 @@ export default Ember.Service.extend({
                 headers: {
                     Authorization: `Bearer ${accessToken}`
                 },
-                success: (data) => resolve(data),
-                error: (_, __, error) => reject(error)
+                error: (_, __, error) => reject(error),
```

Comment:
What is the advantage of using resolve and reject this way? Shouldn't `Ember.$.ajax` already be returning a promise, without creating another one? (or at least a promise-like "deferred" object)

If possible, use promise semantics rather than `success` and `failure` callbacks- it's a bit more consistent with Ember's interface throughout, and makes it easier to reason about chained events.

http://api.jquery.com/jQuery.ajax/#jqXHR